mock/py/mockbuild/package_manager.py

```diff
@@ -178,6 +178,15 @@ def get_pkg_manager_config(self):
         else:
             return self.config['yum.conf']
 
+    @traceLog()
+    def get_pkg_manager_vars(self):
```

Comment:
The 'alias_spec' you specified should cause that this wrapper isn't needed, IIRC.

Reply:
Ping on this.